src/engine-actions/recover-wallet.js

```diff
@@ -19,44 +21,39 @@ async function isValidSeed (seed) {
 }
 
 /**
- * Creates a wallet
+ * Recover wallet funds (on-chain and channel backups)
  *
  * @param {string} password - wallet password, used to unlock lnd wallet
  * @param {Array<string>} seed - 24 word cipher seed mnemonic
- * @param {Buffer} backup - Buffer of the LND static channel backup file
- * @param {?number} recoveryWindow - number in blocks for look back
- * @returns {Promise<Array<string>>} 24 word cipher seed mnemonic
+ * @param {boolean} backup - Buffer of the LND static channel backup file
```

Comment:
This doc is wrong - it's no longer a buffer. Also a name like `useChannelBackup` might be more descriptive for its current usage.

Reply:
I dig. Good suggestion